web/readme.md

```diff
@@ -10,27 +10,23 @@ https://flutter.github.io/samples/#?platform=web.
 
 ## Building samples code
 
-Update Flutter and enable web support
+Building web samples requires Flutter 2.0 or later.
```

Comment:
Is this something we need to say? Flutter 2.0 is stable channel now, I suspect we can assume that people looking at samples are expecting our samples expect the current SDK. 

Otherwise I need to add this disclaimer to pretty much all of our samples and codelabs =)

Reply:
Right! I have removed this line now.